### PR TITLE
Move `act` systems into `Update` together with `input` systems.

### PR DESCRIPTION
**Pull request by seivan**
Also moved `input` systems. 
Doesn't need `lerp` for now.

### CODE REVIEW

**Comment by idanarye** [comment]
Please explain? Why shouldn't they be in the `FixedUpdate` schedule? You've argued in #3 that behavioral systems should use fixed timesteps.

File: examples/example.rs [hunk 1]
```diff
@@ -16,24 +16,25 @@ fn main() {
         .add_systems(Startup, setup)
         .add_systems(
             FixedUpdate,
+            ((
```
Comment (idanarye) [inline examples/example.rs:19]: We should drop one level of tuple here

**Comment by seivan** [comment]
> Please explain? Why shouldn't they be in the `FixedUpdate` schedule? You've argued in #3 that behavioral systems should use fixed timesteps.

Because you're mutating rendered objects directly in the system. 
That's why you'd need `lerp` if you'd had a slower tick rate for `FixedUpdate`, but I am guessing the default one matches the frame rate, it's not really visible.

File: examples/example.rs (hunk 1 above)
Comment (seivan) [reply examples/example.rs]: Done

**Pull request merged**